cytotable/convert.py

```diff
@@ -285,10 +295,10 @@ def _source_chunk_to_parquet(
         source: Dict[str, Any]
             Contains the source data to be chunked. Represents a single
             file or table of some kind along with collected information about table.
-        chunk_size: int
-            Row count to use for chunked output.
-        offset: int
-            The offset for chunking the data from source.
+        page_keys: str:
+            The table and column names to be used for key pagination.
```

Comment:
I don't see `page_keys` in the list of arguments to this function or any references to it in the body.

Reply:
Thanks great catch, I'll update accordingly!